containers/Dockerfile

```diff
@@ -0,0 +1,17 @@
+From python:3.7
+
+RUN apt-get update
+RUN apt-get -y upgrade
+
+RUN apt -y install libblas-dev
+RUN apt -y install liblapack-dev
+RUN apt -y install libgl1-mesa-glx
+
+WORKDIR /
+ENV SCILPY_VERSION="master"
```

Comment:
Why downloading a zip instead of doing a git clone to take automatically the master ?

Reply:
I merge. We can fix it later if necessary. Thanks !